internal/services/automation/automation_dsc_configuration_resource_test.go

```diff
@@ -61,7 +61,8 @@ func TestAccAutomationDscConfiguration_complete(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
+		// not respond by Get API
+		data.ImportStep("log_progress", "hash", "source_version"),
```

Comment:
In this case, we'll need to add a Swagger issue about this and remove those fields from the Schema until these are returned.